<?xml version="1.0" encoding="UTF-8" standalone="yes"?>
<ns2:test-suite start="1519670255046" stop="1519670259700" version="1.5.2" xmlns:ns2="urn:model.allure.qatools.yandex.ru">
    <name>PruebaInicio : Nexus 7 - GMALV[UID=05b6d672, DATA=false, DEVICE=Nexus 7, USER=367]</name>
    <title>PruebaInicio : Nexus 7 - GMALV[UID=05b6d672, DATA=false, DEVICE=Nexus 7, USER=367]</title>
    <test-cases>
        <test-case start="1519670259669" stop="1519670259674" status="broken">
            <name>setup</name>
            <description>@BeforeClass</description>
            <failure>
                <message>NullPointerException: null</message>
                <stack-trace>java.lang.NullPointerException
	at ProyectoMovil.Automatizacion.Utilitario.AbstractTest.setup(AbstractTest.java:151)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
	at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:510)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:211)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:138)
	at org.testng.internal.TestMethodWorker.invokeBeforeClassMethods(TestMethodWorker.java:170)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:104)
	at org.testng.TestRunner.privateRun(TestRunner.java:774)
	at org.testng.TestRunner.run(TestRunner.java:624)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:359)
	at org.testng.SuiteRunner.access$000(SuiteRunner.java:39)
	at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:393)
	at org.testng.internal.thread.ThreadUtil$2.call(ThreadUtil.java:64)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
</stack-trace>
            </failure>
            <steps/>
            <attachments/>
            <labels>
                <label name="feature" value="Ejecucion de pruebas exploratorias iniciales"/>
                <label name="host" value="OCTI107.local"/>
                <label name="thread" value="12688@OCTI107.local.TestNG(14)"/>
            </labels>
        </test-case>
        <test-case start="1519670259682" stop="1519670259685" status="canceled">
            <name>EjecutarNavegacionGeneral</name>
            <title>Probar navegacion</title>
            <description type="text">Prueba general de navegacion de la aplicaci√≥n ordenadas por pasos</description>
            <failure>
                <message>: Skipped due to dependency on other skipped or failed test methods</message>
                <stack-trace>ru.yandex.qatools.allure.testng.AllureTestListener$1: Skipped due to dependency on other skipped or failed test methods
	at ru.yandex.qatools.allure.testng.AllureTestListener.fireTestCaseCancel(AllureTestListener.java:424)
	at ru.yandex.qatools.allure.testng.AllureTestListener.onTestSkipped(AllureTestListener.java:185)
	at org.testng.internal.Invoker.runTestListeners(Invoker.java:1678)
	at org.testng.internal.Invoker.runTestListeners(Invoker.java:1670)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:696)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:816)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1124)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:774)
	at org.testng.TestRunner.run(TestRunner.java:624)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:359)
	at org.testng.SuiteRunner.access$000(SuiteRunner.java:39)
	at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:393)
	at org.testng.internal.thread.ThreadUtil$2.call(ThreadUtil.java:64)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
</stack-trace>
            </failure>
            <steps/>
            <attachments/>
            <labels>
                <label name="testSuite" value="PruebaInicio"/>
                <label name="testGroup" value="Nexus 7 - GMALV"/>
                <label name="testClass" value="ProyectoMovil.Automatizacion.Ejecutar.Pruebas.EjecucionInicioPrueba"/>
                <label name="testMethod" value="EjecutarNavegacionGeneral"/>
                <label name="severity" value="normal"/>
                <label name="story" value="Probar navegacion"/>
                <label name="feature" value="Ejecucion de pruebas exploratorias iniciales"/>
                <label name="host" value="OCTI107.local"/>
                <label name="thread" value="12688@OCTI107.local.TestNG(14)"/>
            </labels>
        </test-case>
        <test-case start="1519670259685" stop="1519670259685" status="canceled">
            <name>setOut</name>
            <description>@AfterClass</description>
            <failure>
                <message>: Skipped due to dependency on other skipped or failed test methods</message>
                <stack-trace>ru.yandex.qatools.allure.testng.AllureTestListener$1: Skipped due to dependency on other skipped or failed test methods
	at ru.yandex.qatools.allure.testng.AllureTestListener.fireTestCaseCancel(AllureTestListener.java:424)
	at ru.yandex.qatools.allure.testng.AllureTestListener.onConfigurationSkip(AllureTestListener.java:119)
	at org.testng.internal.Invoker.runConfigurationListeners(Invoker.java:1656)
	at org.testng.internal.Invoker.handleConfigurationSkip(Invoker.java:253)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:191)
	at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:138)
	at org.testng.internal.TestMethodWorker.invokeAfterClassMethods(TestMethodWorker.java:220)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:111)
	at org.testng.TestRunner.privateRun(TestRunner.java:774)
	at org.testng.TestRunner.run(TestRunner.java:624)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:359)
	at org.testng.SuiteRunner.access$000(SuiteRunner.java:39)
	at org.testng.SuiteRunner$SuiteWorker.run(SuiteRunner.java:393)
	at org.testng.internal.thread.ThreadUtil$2.call(ThreadUtil.java:64)
	at java.util.concurrent.FutureTask.run(FutureTask.java:266)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1149)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:624)
	at java.lang.Thread.run(Thread.java:748)
</stack-trace>
            </failure>
            <steps/>
            <attachments/>
            <labels>
                <label name="feature" value="Ejecucion de pruebas exploratorias iniciales"/>
                <label name="host" value="OCTI107.local"/>
                <label name="thread" value="12688@OCTI107.local.TestNG(14)"/>
            </labels>
        </test-case>
    </test-cases>
    <labels>
        <label name="framework" value="TestNG"/>
        <label name="language" value="JAVA"/>
    </labels>
</ns2:test-suite>
